src/Components/Grants/GrantEdit.jsx

```diff
@@ -70,6 +42,12 @@ export default function GrantEdit(props) {
     });
   };
 
+  const handleDelete = () => {
+    if (confirm(`Are you sure you want to delete this grant?`)) {
```

Comment:
We'll follow up with custom popups in the future
```suggestion
    // eslint-disable-next-line no-restricted-globals
    if (confirm(`Are you sure you want to delete this grant?`)) {
```